lib/session/session.go

```diff
@@ -106,6 +106,8 @@ type Session struct {
 	AppName string `json:"app_name"`
 	// Owner is the name of the session owner, ie the one who created the session.
 	Owner string `json:"owner"`
+	// IsModeratedSession is true if the session requires moderation.
+	IsModeratedSession bool `json:"isModeratedSession"`
```

Comment:
Minor: everything else in this structure is using underscores, why is this camel case?

Reply:
Was using my frontend brain when I did this part. Thanks for the catch